cloth-diapers/cloth-diapers-basics.md

```diff
@@ -0,0 +1,296 @@
+size of modern cloth diaper that **will accompany a child throughout the
+majority of their diaper journey**. They are marketed as fitting babies from
+around 3-15kg (with a small amount of variation per brand) but usually don't fit
+newborns because the leg holes are designed for slightly bigger babies.
++ **XL:** Nomenclature varies but these are often designated 'XL' or 'Super'
+and are made for older toddlers but sizes vary widely. They could refer to
+diapers that are 9 - 18kgs for example or even diapers that are 15 - 20kgs, so
+it's important to review the stipulations of fit from the individual brand.
+
+**One size (OS) are by _far_ the most common**, with many people deciding to
+wait until their babies fit into this size before they start their cloth diaper
+journey.
+
+There _are_ 'newborn hacks' of folds that allow you to use OS diapers on smaller
+babies but these are greatly dependent on the baby's body and on the diaper
+brands.
+
+# Our Choices
+
+Our choices have evolved over time, not only as we have gained experience
+but also as little A has grown and had different requirements.
+
+## Diaper Sizes
+I chose to invest in a small stash of newborn cloth diapers because I wanted
+to start as soon as possible. I wanted to learn the systems and understand
+what we preferred to use _before_ buying a full stash of OS diapers that we
+might hate because they were the wrong type or materials.
+
+Unless you don't plan to start cloth diapering until baby is 3 months old or
+so, I think this is a good investment because it lets you learn the ropes and
+they hold their value extremely well (more so than OS) so can be sold on easily.
+
+
+## Diaper Types
+Now that we have 'dialed things in' a little more, there are three types of
+diapers we use (there are others, but these are the only 3 in our rotation):
+
+### 1. All-In-Ones (AIO)
+This is our 'favourite' kind of diaper because it requires the **_least_ time
+to put together** by _far_. For the majority you just have to push the 'tongue'
+(sewn in insert) back into the pocket and that's it.
+With a small number (in our stash it's the Motherease Wizard Unos and
+Bellz Bumz Reusabelles), they're ready to go straight off the line.
+
+Whilst we now find we have to reinforce the majority of these with an extra
+3 layer bamboo insert, this isn't the case with all of them.
+
+**The biggest issue with AIOs** is that they take a lot longer to dry.
+
+**The key thing to be aware of** is that not all AIOs are created equally
+in terms of absorption. It is _very important_ to understand what the material
+composition of these diapers is so that you know what you're dealing with.
+
+For example:
+[Modern Cloth Nappies AIOs](https://www.modernclothnappies.co.uk/collections/pearl-pocket-all-in-one-nappies/products/pearl-pocket-all-in-one-reusable-one-size-nappy)
+(my absolute favourite AIOs from our stash), have a super absorbent
```

Comment:
```suggestion
(my absolute favourite AIOs from our set), have a super absorbent
```
Have you been reading the blog of a mother who was a reformed drug dealer/user? 🤷‍♂️

Reply:
Again, I'm not changing this, but adding a terminology section #36 .
This is a community word that it pays to be familiar with.